src/rust/src/lazy/dataframe.rs

```diff
@@ -70,6 +70,14 @@ impl LazyFrame {
         )
     }
 
+    fn tail(&self, n: f64) -> List {
+        r_result_list(
+            try_f64_into_u32(n)
+                .map(|n| LazyFrame(self.0.clone().tail(n)))
+                .map_err(|err| format!("tail: {}", err)),
+        )
+    }
+
```

Comment:
this can now be expressed in a more ergonomic way. But essentially the same

robj_to! converts or write a suitable err msg, it also adds in the name of argument which failed

when using robj_to! I think it is fine to just skip testing of error handling on every single method, robj_to! is already tested plenty.

```rust
fn tail(&self, n: Robj) -> Result<LazyFrame, String> {
        Ok(LazyFrame(self.0.clone().tail(robj_to!(u32, n)?)))
}
```
err example
```R
> pl$DataFrame(iris)$lazy()$tail(n=-3)

Error: the arg [n] the value -3 cannot be less than zero
when calling :
pl$DataFrame(iris)$lazy()$tail(n = -3)
```

Reply:
thanks, this is a very nice shortcut.